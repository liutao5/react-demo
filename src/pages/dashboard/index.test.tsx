import React from 'react';
import renderer from 'react-test-renderer';
import Dashboard from './index';

test('test Dashboard page', () => {
  const component = renderer.create(
    <Dashboard />,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
