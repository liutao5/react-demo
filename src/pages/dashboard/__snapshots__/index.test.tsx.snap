// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Dashboard page 1`] = `"dashboard"`;
